backend/server/controllers/weeks.js

```diff
@@ -46,7 +46,8 @@ module.exports = {
       if (week) {
         const updatedChecks = req.body.checks || {}
         await week.update({
-          points: req.body.points || week.points,
+          points: req.body.points || week.points || null,
+          grade: req.body.grade || null,
```

Comment:
Wouldn't this set `grade` to null if request body doesn't include it?

Reply:
Yes, but it always should - is there a case when the request body does not include it?

Reply:
Currently no, but in the future it could be possible and I think it would be confusing if the request behaves differently for `grade` than other properties.

Reply:
The issue is that it should be possible to remove the grade from an existing review

Reply:
In that case we can set `grade` to `null` in the request?

Reply:
...which wouldn't work with `||`, but we could use ternary syntax somehow if it is really necessary

Reply:
Not really necessary, I usually just overthink about how to handle possible future cases. In any case, we should probably make sure that all backend endpoints behave in a consistent way, e.g. how to handle missing properties, should deletion use POST or DELETE request etc.